config/syncer-control/role.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
is this the role applied in the control plane or the data plane?

Reply:
ah I see it seems to be the control plane one. Looks good. shame we can't limit it to just annotation updates

Reply:
Yes I checked if there was something similar to `/status` for annotations but didn't find anything